main.go

```diff
@@ -746,7 +754,10 @@ func usage() {
 	put("    dd\tDelete file or dir")
 	_ = w.Flush()
 	_, _ = fmt.Fprintf(os.Stderr, "\n")
-	os.Exit(1)
```

Comment:
Let's return this.

Reply:
Is there a reason to use `os.Exit` instead of returning from the main function?

Using it prints to the terminal something like "program exited with status code ..."

Reply:
Yes. As exit code 1 will break cmd substitution. And next is possible: ll -h

Reply:
Understood, thanks for the insight